From 4734ff39a22efbfe58c1758b19933aa770be84f0 Mon Sep 17 00:00:00 2001
From: Chenhui Zhao <chenhui.zhao@nxp.com>
Date: Mon, 18 Apr 2016 11:56:04 +0800
Subject: [PATCH 1303/1429] arm: dts: ls1021a: add missed the rcpm-wakeup
 property

Added the property of rcpm-wakeup for the nodes, gpio0, lpuart0 and
enet0.

Signed-off-by: Chenhui Zhao <chenhui.zhao@nxp.com>
[Original patch taken from QorIQ-SDK-V2.0-20160527-yocto]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/arm/boot/dts/ls1021a.dtsi |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/ls1021a.dtsi b/arch/arm/boot/dts/ls1021a.dtsi
index b0e3506..4792360 100644
--- a/arch/arm/boot/dts/ls1021a.dtsi
+++ b/arch/arm/boot/dts/ls1021a.dtsi
@@ -460,6 +460,7 @@
 			#gpio-cells = <2>;
 			interrupt-controller;
 			#interrupt-cells = <2>;
+			rcpm-wakeup = <&rcpm 0x00000040 0x0>;
 		};
 
 		gpio1: gpio@2310000 {
@@ -562,6 +563,7 @@
 			interrupts = <GIC_SPI 80 IRQ_TYPE_LEVEL_HIGH>;
 			clocks = <&sysclk>;
 			clock-names = "ipg";
+			rcpm-wakeup = <&rcpm 0x0 0x40000000>;
 			status = "disabled";
 		};
 
@@ -725,6 +727,7 @@
 			interrupt-parent = <&gic>;
 			model = "eTSEC";
 			fsl,magic-packet;
+			rcpm-wakeup = <&rcpm 0x80000000 0x0>;
 			ranges;
 			dma-coherent;
 
-- 
1.7.5.4

